addon/components/offering-form.js

```diff
@@ -439,28 +446,40 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
 
     yield this.get('saveOffering').perform();
   }),
-  updateDurationMinutes: task(function * (minutes) {
-    let {validations} = yield this.validate();
-    this.send('addErrorDisplayFor', 'durationMinutes');
 
-    if (validations.get('durationMinutes.isInvalid')) {
+  updateDurationHours: task(function * (hours) {
+    yield timeout(DEBOUNCE_DELAY);
+    this.set('hourBuffer', hours);
+    const { validations } = yield this.validate();
+    const hourValidation = validations.toArray()[2];
```

Comment:
Where does the `[2]` come from? I'm worried it's just the second validation and would change if the ordering changed. If that's true it should be done by name and not index.

Reply:
I don't think that varies. I think it goes by the order of the validations written.

```
  room: [
   ... 
  ],
  numberOfWeeks: {
    ...
  },
  hourBuffer: [
    ...
  ],
  minuteBuffer: [
    ...
  ],
  learnerGroups: {
     ...
  }
```
This order would dictate the array order. I can look again but it wasn't an object when I checked.

Reply:
That's my concern. It varies if you add a validation or move them around.